Samples/MBEDAppShield/MBED.lpr

```diff
@@ -0,0 +1,377 @@
+program MBED;
```

Comment:
Will not yet add this sample as it is looking way too complicated and often does not use concepts of MBF

Reply:
That is ok for me.
Its just a proof of concept.
If all concepts of MBF are clear and stable, this example can easily be adapted.